src/app/ledger/main.c

```diff
@@ -890,11 +904,11 @@ replay( fd_ledger_args_t * args ) {
               --slot-history 5000 --allocator wksp --tile-cpus 5-21 --funk-page-cnt 16
   */
 
-  restore( args ); /* Restores checkpointed workspace(s) */
-
   init_funk( args ); /* Joins or creates funk based on if one exists in the workspace */
   init_blockstore( args ); /* Does the same for the blockstore */
 
+  restore( args ); /* Restores checkpointed workspace(s) */
+
```

Comment:
why do we now restore after. init_funk lets you join a funk which is loaded in

Reply:
unarchive needs the funk to already exist. i can split this up